### PR TITLE
Recipe for 0.2 version of Vatri Google Drive Bundle

### PR DESCRIPTION
**Pull request by vatri**
| Q             | A
| ------------- | ---
| License       | MIT

Recipe for 0.2 version of Vatri Google Drive Bundle

### CODE REVIEW

File: vatri/google-drive-bundle/0.2/manifest.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Vatri\\GoogleDriveBundle\\VatriGoogleDriveBundle": ["all"]
```
Comment (Nyholm) [inline vatri/google-drive-bundle/0.2/manifest.json]: Is this really the correct path?

Comment (vatri) [reply vatri/google-drive-bundle/0.2/manifest.json]: I assume yes. See attached - that's the most recent version of the bundle file.

Also you can check my issue #681. 

Screenshot_2019-06-29_12-32-47

**Comment by Nyholm** [comment]
The CI continues to be red. Im not sure why but I need to see at least one green CI job before I can merge.

**Comment by Nyholm** [comment]
Feel free to make a new PR if you are still interested to finish this.

**Pull request closed without merging**